lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -607,7 +725,7 @@ public static function get_user_global_styles_post_id() {
 			return static::$user_custom_post_type_id;
 		}
 
-		$user_cpt = static::get_user_data_from_wp_global_styles( wp_get_theme(), true );
+		$user_cpt = static::get_user_data_from_wp_global_styles( wp_get_theme() );
```

Comment:
mmm, why do we need this change? The resulting behavior is that no post will be created if none exists.